src/SwedbankPay/Api/Response.php

```diff
@@ -0,0 +1,239 @@
+<?php
+// phpcs:ignoreFile
+
+namespace SwedbankPay\Api;
+
+use SwedbankPay\Api\Client\Exception;
+
+/**
+ * Response class to handle JSON objects
+ *
+ * @note For backwards compatibility for previous versions
+ * @SuppressWarnings(PHPMD.ExcessivePublicCount)
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ */
+class Response implements \ArrayAccess
```

Comment:
Codacy found an issue: [The class Response has 13 non-getter- and setter-methods. Consider refactoring Response to keep number of methods under 10.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6212562)